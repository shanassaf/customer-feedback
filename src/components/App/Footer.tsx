import React from 'react';
import styled from 'styled-components';

const StyledFooter = styled.footer``;

export const Footer = () => (
	<StyledFooter>
		<img src="../images/checkout.jpeg" alt="Checkout logo" />
	</StyledFooter>
);
