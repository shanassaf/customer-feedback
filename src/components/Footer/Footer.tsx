import React from 'react';
import styled from 'styled-components';

const StyledFooter = styled.footer`
	padding: 50px 20px;
	font-size: 14px;
	text-align: center;
`;
const StyledLink = styled.a`
	text-decoration: none;
	color: #800080;

	:hover {
		text-decoration: underline;
	}
`;

export const Footer = (): JSX.Element => (
	<StyledFooter>
		Check out the repository for this webpage on
		<StyledLink href="https://github.com/shanassaf/customer-feedback"> github</StyledLink>
	</StyledFooter>
);
