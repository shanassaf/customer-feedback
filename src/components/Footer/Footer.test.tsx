import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from './Footer';

describe('Footer', () => {
	test('component renders correctly', () => {
		const { asFragment } = render(<Footer />);
		expect(asFragment()).toMatchSnapshot();
	});
});
