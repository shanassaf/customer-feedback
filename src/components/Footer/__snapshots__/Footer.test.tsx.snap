// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly 1`] = `
<DocumentFragment>
  <footer
    class="sc-bdnylx eKwBmK"
  >
    Check out the repository for this webpage on
    <a
      class="sc-gtssRu jDusuy"
      href="https://github.com/shanassaf/customer-feedback"
    >
       github
    </a>
  </footer>
</DocumentFragment>
`;
